modules/social_features/social_post/modules/social_post_photo/social_post_photo.install

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * @file
+ * The Social post photo installation.
+ */
+
+
+function social_post_photo_uninstall() {
+  $configs = [
```

Comment:
It is best to change the existing posts of bundle 'photo' to the old 'post' bundle again on uninstall. On install: the other way around.